pom.xml

```diff
@@ -52,7 +53,110 @@
 		</pluginRepository>
 	</pluginRepositories>
 
-    <properties>
-        <openmrsPlatformVersion>2.6.0</openmrsPlatformVersion>
-    </properties>
-</project>
+	<properties>
+		<openmrsPlatformVersion>2.6.0</openmrsPlatformVersion>
+		<webservices.restVersion>2.44.0</webservices.restVersion>
+	</properties>
+	<dependencyManagement>
+		<dependencies>
+		
+			<dependency>
+				<groupId>org.mockito</groupId>
+				<artifactId>mockito-junit-jupiter</artifactId>
+				<version>3.12.4</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.mockito</groupId>
+				<artifactId>mockito-core</artifactId>
+				<version>4.0.0</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.mockito</groupId>
+				<artifactId>mockito-inline</artifactId>
+				<version>3.9.0</version>
+				<scope>test</scope>
+			</dependency>
+
+			<dependency>
+				<groupId>org.powermock</groupId>
+				<artifactId>powermock-module-junit4</artifactId>
+				<version>2.0.9</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.powermock</groupId>
+				<artifactId>powermock-api-mockito2</artifactId>
+				<version>2.0.9</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>junit</groupId>
+				<artifactId>junit</artifactId>
+				<version>4.13.1</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.openmrs.web</groupId>
+				<artifactId>openmrs-web</artifactId>
+				<scope>provided</scope>
+				<version>${openmrsPlatformVersion}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.openmrs.web</groupId>
+				<artifactId>openmrs-web</artifactId>
+				<scope>provided</scope>
+				<version>${openmrsPlatformVersion}</version>
+				<classifier>tests</classifier>
```

Comment:
This should also be scoped as provided or test

Reply:
@ibacher are you referring to line 110?

Reply:
Whoops! Missed that!